From ee1b99f962fd72b8c7a566b972c21724c6a2b6a3 Mon Sep 17 00:00:00 2001
From: Abhijith Desai <abdesai@qti.qualcomm.com>
Date: Wed, 3 Jun 2026 03:34:32 -0700
Subject: [PATCH] Convert GBM backend shared module and update installation
 rules.

Convert GBM backend shared module and update installation rules,
Update installation rules to only deploy unversioned msm_gbm.so

Signed-off-by: Abhijith Desai <abdesai@qti.qualcomm.com>

Forwarded: https://github.com/qualcomm-linux/gbm-msm-backend/commit/4191b72b984ff52188f0682b918309e72ea78971
---
 src/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/src/meson.build
+++ b/src/meson.build
@@ -22,7 +22,7 @@
     'schema_parser.h',
 ]
 
-msm_gbm = library('msm_gbm',
+msm_gbm = shared_module('msm_gbm',
     src,
     dependencies : [
         gbm,
@@ -31,8 +31,8 @@
         libdl,
     ],
     include_directories : ext_includes,
-    version : meson.project_version(),
     install : true,
+    install_dir: libdir / 'gbm',
     name_prefix: '',
     c_args: c_args,
 )
